src/popup/snooze.scss

```diff
@@ -1,29 +1,32 @@
 @import 'node_modules/rc-calendar/assets/index';
 @import 'node_modules/rc-time-picker/assets/index';
+@import 'node_modules/photon-colors/colors.scss';
 
 $panel-width: 320px;
 $panel-shadow-width: 13px;
+$white: #fff;
```

Comment:
We don't have `white` and `black` in the photon-colors?!?

Reply:
apparently not :)